interview-questions.md

```diff
@@ -9,59 +9,79 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The 'this' in JavaScript is relates to what 'this' is trying to access, such as this.state."enter prop name here". We use 'this' to access that property and use it throughout our code
 
-  Researched answer:
+  Researched answer: The 'this' keyword references a Javascript element depending on the scope or context of its use. 
```

Comment:
More specifically, this references an object.  For instance, anything inside of state is an object and is why we would call this.state to access the values within state.